.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501, E302, W503
```

Comment:
These error codes come from a few places, but they represent a specific rule violation. `flake8` inherits a bunch of [rules from pycodestyle](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes) (which all begin with E and W), and extends that list with a [bunch of its own rules](https://flake8.pycqa.org/en/latest/user/error-codes.html#error-violation-codes) that begin with F.